modules/oe_content_news/config/post_updates/00003_update_form_display/core.entity_form_display.node.oe_news.default.yml

```diff
@@ -0,0 +1,260 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.node.oe_news.body
+    - field.field.node.oe_news.oe_author
+    - field.field.node.oe_news.oe_news_featured_media
+    - field.field.node.oe_news.oe_news_location
+    - field.field.node.oe_news.oe_publication_date
+    - field.field.node.oe_news.oe_reference_code
+    - field.field.node.oe_news.oe_related_links
+    - field.field.node.oe_news.oe_subject
+    - field.field.node.oe_news.oe_summary
+    - field.field.node.oe_news.oe_teaser
+    - node.type.oe_news
+  module:
+    - datetime
+    - field_group
+    - link
+    - maxlength
+    - path
+    - rdf_skos
+    - text
+third_party_settings:
+  field_group:
+    group_alternative_titles_teaser:
+      children:
+        - oe_content_short_title
+        - oe_content_navigation_title
+        - oe_teaser
+      parent_name: ''
+      weight: 6
+      format_type: details
+      region: content
+      format_settings:
+        id: ''
+        classes: ''
+        description: ''
+        open: true
+        required_fields: true
+      label: 'Alternative titles and teaser'
+    group_details:
```

Comment:
I would not add this group here, such grouping is done on ewcms side (if I understood correctly). We should only add groups for fields that belong together like an event name and date fields, or venue name and location, so on.

Reply:
I did it as in Project CT - there are groups of fields.